ONSdigital/eq-terraform
# Disable root ebs delete_on_termination

## What is the context of this PR?
[trello](https://trello.com/c/JnsvipYm/1009-disable-delete-on-terminate-for-rabbit-volume)

This won't have any effect on existing instances. See note on
https://www.terraform.io/docs/providers/aws/r/instance.html.

To manually apply to an existing instance run `aws ec2
modify-instance-attribute --instance-id i-XXXXXXXXXXXX
--block-device-mappings "[{\"DeviceName\":
\"/dev/sda1\",\"Ebs\":{\"DeleteOnTermination\":false}}]"`

### How to review
Deploy rabbit infrastructure and check `delete on termination` flag
1. Go to EC2 console
1. Click on RabbitMQ EC2 instance
1. Click on `Root device` in description tab
1. Observe `delete on termination` status
